package.json

```diff
@@ -15,7 +15,7 @@
     "load-addresses": "yarn workspace @uma/core load-addresses",
     "create-release": "lerna version --no-git-tag-version --conventional-commits --no-push",
     "publish-release": "./scripts/run_release.sh",
-    "qbuild": "yarn workspace @uma/common build && yarn workspace @uma/core build && yarn lerna run build --stream --ignore '*/*dapp*' --ignore '@uma/common' --ignore '@uma/core'",
+    "qbuild": "yarn lerna run build --stream",
```

Comment:
We can now simplify the build command because there are no circular deps!

Reply:
v exciting